Diagnostics/FreeBusyChecker/FreeBusyChecker.ps1

```diff
@@ -0,0 +1,303 @@
+        } else {
+            $Script:ExchangeOnPremEWS = ($EWSVirtualDirectory.externalURL.AbsoluteUri)
+        }
+    } else {
+        $Script:ExchangeOnPremEWS = ($OnPremEWSUrl)
+    }
+
+    if (-not $OnPremDomain) {
+        $ADDomain = Get-ADDomain
+        $Script:ExchangeOnPremLocalDomain = $ADDomain.forest
+    } else {
+        $Script:ExchangeOnPremLocalDomain = $OnPremDomain
+    }
+
+    $Script:ExchangeOnPremLocalDomain = $ADDomain.forest
+    if ([string]::IsNullOrWhitespace($ADDomain)) {
+        $Script:ExchangeOnPremLocalDomain = $exchangeOnPremDomain
+    }
+
+    if ($ExchangeOnPremDomain) {
+        $Script:FedInfoEOP = Get-federationInformation -DomainName $ExchangeOnPremDomain  -BypassAdditionalDomainValidation -ErrorAction SilentlyContinue -WarningAction SilentlyContinue | Select-Object *
+    }
+    #endregion
+
+    if ($Help) {
+        PrintDynamicWidthLine
+        ShowHelp
+        PrintDynamicWidthLine
+        exit
+    }
+    #region Show Parameters
+    $Script:IntraOrgCon = Get-IntraOrganizationConnector -WarningAction SilentlyContinue -ErrorAction SilentlyContinue | Select-Object Name, TarGetAddressDomains, DiscoveryEndpoint, Enabled
+    ShowParameters
+    CheckParameters
+    if ($Script:IntraOrgCon.enabled -Like "True") {
+        $Auth = hostOutputIntraOrgConEnabled($Auth)
+    }
+    if ($Script:IntraOrgCon.enabled -Like "False") {
+        hostOutputIntraOrgConNotEnabled
+    }
+    # Free busy Lookup methods
+    PrintDynamicWidthLine
+    $Script:OrgRel = Get-OrganizationRelationship | Where-Object { ($_.DomainNames -like $Script:ExchangeOnlineDomain) }  -WarningAction SilentlyContinue -ErrorAction SilentlyContinue  | Select-Object Enabled, Identity, DomainNames, FreeBusy*, TarGet*
+    $Script:EDiscoveryEndpoint = Get-IntraOrganizationConfiguration -WarningAction SilentlyContinue -ErrorAction SilentlyContinue | Select-Object OnPremiseDiscoveryEndpoint
+    $Script:SPDomainsOnprem = Get-SharingPolicy -WarningAction SilentlyContinue -ErrorAction SilentlyContinue | Format-List Domains
+    $Script:SPOnprem = Get-SharingPolicy  -WarningAction SilentlyContinue -ErrorAction SilentlyContinue | Select-Object *
+
+    if ($Org -contains 'ExchangeOnPremise' -or -not $Org) {
+        #region DAuth Checks
+        if ($Auth -like "DAuth" -OR -not $Auth -or $Auth -like "All") {
+            Write-Host $TestingDAuthConfiguration
+            OrgRelCheck -OrgRelParameter $Script:OrgRel
+            PrintDynamicWidthLine
+            FedInfoCheck
+            FedTrustCheck
+            AutoDVirtualDCheck
+            PrintDynamicWidthLine
+            EWSVirtualDirectoryCheck
+            AvailabilityAddressSpaceCheck
+            TestFedTrust
+            TestOrgRel
+        }
+        #endregion
+        #region OAuth Check
+        if ($Auth -like "OAuth" -or -not $Auth -or $Auth -like "All") {
+            Write-Host $TestingOAuthConfiguration
```

Comment:
Empty variable

Reply:
Changed to Text Output

![image](https://github.com/user-attachments/assets/df3452f4-ff73-4b61-832c-8df587e54d45)